corona-build-and-test.yml

```diff
@@ -35,6 +36,23 @@ stages:
     # A true statement is expected to allow jobs to run. Here is the default.
     - when: on_success
 
+is-machine-up-check:
```

Comment:
Is it possible to consolidate this step in one file and re-use it for any machine rather than repeating it in each platform file?